.rubocop.yml

```diff
@@ -0,0 +1,104 @@
+# Configuration hierarchy:
+#
+# 1. Rubocop defaults
+# 2. Carbon Five defaults (this file)
+# 3. Project overrides
+#
+# See http://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-remote-url for details.
+#
```

Comment:
I think we should remove this since it only relevant to the `inherit_from:` directive, which we are not using here.